chain-impl-mockchain/src/testing/extensions/mod.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{accounting::account::AccountState, value::Value};
+
+impl AccountState<()> {
```

Comment:
Maybe use a different name than the type under test to reduce ambiguity. `AccountStateValidator`?

Reply:
Actually it's not needed anymore